public/build/build.css

```diff
@@ -0,0 +1,15 @@
+
+span.plagiarism-similarity-score[data-v-e6695da2] {
```

Comment:
Where does the hard-coded hash come from? (Will it change?)

Reply:
Thats result of the <styles scoped>. To scope css styling for given component.

Reply:
And yes, it will change every build.